packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/testSetItemSelectionAtTo.st

```diff
@@ -0,0 +1,19 @@
+testing
+testSetItemSelectionAtTo
+
+	| changeListItems |
+	changeListItems := self createDummyChangeListItems: 2 with: false.
+	(changeListItems at: 2) isSelected: true.
+	self changesBrowser changeListItems: changeListItems.
+	
+	self assert: (self changesBrowser isItemSelectedAt: 1) equals: false.
+	self assert: (self changesBrowser isItemSelectedAt: 2) equals: true.
+	self changesBrowser setItemSelectionAt: 1 to: true.
+	self assert: (self changesBrowser isItemSelectedAt: 1) equals: true.
+	self assert: (self changesBrowser isItemSelectedAt: 2) equals: true.
+	
+	self assert: (self changesBrowser isItemSelectedAt: 1) equals: true.
```

Comment:
Why do you check those twice?

Reply:
Well good question. We thought of these as kind-of two different test cases, separated by the newline. We should probably refactor this into two separate tests or remove an assert